README.rst

```diff
@@ -85,3 +85,69 @@ testing with live email addresses.  It works like this:
     # Destroy the account once you're done with it.
     acct.clear()
     client.destroy_account(acct.email, "MySecretPassword")
+
+
+Using Firefox Account BrowserID with Requests
+=============================================
+
+You can use the ``FxABrowserIdAuth`` to build the BrowserId assertion:
+
+.. code-block:: python
+
+    from fxa.core import Client
+    from fxa.requests import FxABrowserIdAuth
```

Comment:
What do you think about moving this into a sub-package like `fxa.plugins` or similar, to make it more obvious the scope of what's offered?